### PR TITLE
Fix LoginSettings deployment migration

### LINKED ISSUE #14095: LoginSettings not found in Assembly Orchard.Users
### Describe the bug
I have a deployment step "Login Settings", and after updating Orchard from 1.6.0 to 1.7.0, I'm getting this exception when trying to load the list od Deployment plans:

```
Newtonsoft.Json.JsonSerializationException: Error resolving type specified in JSON 'OrchardCore.Settings.Deployment.SiteSettingsPropertyDeploymentStep`1[[OrchardCore.Users.Models.LoginSettings, OrchardCore.Users]], OrchardCore.Settings.Core'. Path 'DeploymentSteps[6].$type', line 1, position 1332.
 ---&gt; Newtonsoft.Json.JsonSerializationException: Could not find type 'OrchardCore.Settings.Deployment.SiteSettingsPropertyDeploymentStep`1[[OrchardCore.Users.Models.LoginSettings, OrchardCore.Users]]' in assembly 'OrchardCore.Settings.Core, Version=1.7.0.0, Culture=neutral, PublicKeyToken=null'.
 ---&gt; Newtonsoft.Json.JsonSerializationException: Could not find type 'OrchardCore.Users.Models.LoginSettings' in assembly 'OrchardCore.Users, Version=1.7.0.0, Culture=neutral, PublicKeyToken=null'.
```

How can I resolve this issue?

### PR DESCRIPTION
**Pull request by MikeAlhayek**
Fix #14095

@sebastienros if you approve this soon, we may be able to release 1.7 Thursday.

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Users/Migrations.cs [hunk 1]
```diff
@@ -240,5 +243,50 @@ public int UpdateFrom11()
...
+
+                try
+                {
+                    logger.LogDebug("Updating User Settings");
+
+                    var quotedTableName = dialect.QuoteForTableName(table, session.Store.Configuration.Schema);
+                    var quotedContentColumnName = dialect.QuoteForColumnName("Content");
+                    var quotedTypeColumnName = dialect.QuoteForColumnName("Type");
+
+                    var updateCmd = $"UPDATE {quotedTableName} " +
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Users/Migrations.cs:272]: It's ok to use a single line, and if you are adventurous you can even try the new `"""` raw string syntax, might be the best option even here. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Users/Migrations.cs:272]: But it will probably not work with templated strings :/

Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.Users/Migrations.cs]: `"""` is not available in C# 10.

I created a mile long SQL command :) any other suggestions?

**Review by sebastienros — APPROVED** [review]

**Comment by sebastienros** [comment]
We will probably remove net6.0 once net8.0 is shipped.

**Pull request merged**